test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,42 @@
+#include "TutorialServo.hpp"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float
+  minPulsewidthInMs, float maxPulsewidthInMs) :
+  m_servoPwmOut(servoPin),
+  m_servoRangeInDegrees(servoRangeInDegrees),
+  m_minPulsewidthInMs(minPulsewidthInMs),
+  m_maxPulsewidthInMs(maxPulsewidthInMs)
+{
+  m_servoPwmOut.period_ms(20);
+}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+  
+  if(degrees > m_servoRangeInDegrees) {
+      m_servoPwmOut.pulsewidth(m_maxPulsewidthInMs);
+  }
+  else if (degrees < 0 ) {
+      m_servoPwmOut.pulsewidth(m_minPulsewidthInMs);
+  }
+  else{
+      m_servoPwmOut.pulsewidth(m_minPulsewidthInMs + degrees/(m_servoRangeInDegrees * 1000));
```

Comment:
Think about this mapping again, would we also need to consider `m_maxPulseWidthInMs`? Also, `PwmOut.pulsewidth()` accepts a pulsewidth in seconds, not miliseconds